core/deno_cache.ts

```diff
@@ -125,7 +125,7 @@ export class CacheModule implements DenoCacheModule {
       targetOriginDir = path.join(
         getDenoDepsDir(),
         url.protocol.replace(/:$/, ""), // https: -> https
-        url.hostname
+        url.hostname + (url.port ? `_PORT${url.port}` : url.port)
```

Comment:
```suggestion
        `${url.hostname}${url.port ? `_PORT${url.port}` : ""}`
```

@lucacasonato Thanks for finding this!

Also, not sure if the Deno linter has a rule about using the addition operator for string concatenation, but in any case, I think consistency is good for this line and https://github.com/denoland/vscode_deno/pull/173/files#diff-f5bbeb1e4c5f2facaa107bf559832939R78